db/models.py

```diff
@@ -50,3 +57,52 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="unique_row_seat_movie_session_ticket"
+            )
+        ]
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                "row": [
+                    "row number must be in available range: (1, rows): (1, {})"
+                    .format(self.movie_session.cinema_hall.rows)
+                ]
+            })
+
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError({"seat": [
+                "seat number must be in available range: "
+                "(1, seats_in_row): (1, {})"
+                .format(self.movie_session.cinema_hall.seats_in_row)
```

Comment:
It would be better to not place `.format` on a new line and leave it on the same line with string, performing line breaking after `(`